notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<p>Add a link to the monitoring console.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>